openpype/hosts/houdini/plugins/publish/validate_output_node.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import PublishValidationError
+from openpype.hosts.houdini.api.action import (
+    SelectInvalidAction,
+    SelectROPAction,
+)
+
+import hou
+
+
+class ValidateOutputNode(pyblish.api.InstancePlugin):
```

Comment:
I feel like this class name and plug-in label is a bit too generic for an FBX specific check for validating whether the output node is either SOP or ObjNode. Should we be more specific to avoid name clashes in the future?

Reply:
> Should we be more specific to avoid name clashes in the future?
100%  agreed! 

tbh, I don't know what to call it, any recommendations ?

Reply:
We could just do `ValidateFBXOutputNode` to make it specific to the family?